fabfile.py

```diff
@@ -847,12 +961,38 @@ def install_assembl_deps():
     """
     Will install commonly needed build deps for pip django virtualenvs.
     """
-    sanitize_env()
-    execute(skeleton_env, None)
     execute(install_basetools)
     execute(install_builddeps)
 
 
+@task
+def install_certbot():
+    """Install letsencrypt.org certbot"""
+    if env.mac:
+        return
+    if exists('/etc/os-release'):
+        release_data = run('cat /etc/os-release')
+        if 'jessie' in release_data:
+            sudo("echo 'deb http://ftp.debian.org/debian jessie-backports main' >> /etc/apt/sources.list")
+            sudo("apt-get update")
+        elif 'ubuntu' in release_data:
+            sudo("apt-get install software-properties-common")
+            sudo("add-apt-repository ppa:certbot/certbot")
+            sudo("apt-get update")
+        else:
+            raise NotImplementedError("Unknown distribution")
+        sudo("apt-get install python-certbot-nginx")
```

Comment:
In newer version of Ubuntu (16.01) and later, it's `apt-get install letsencrypt`. Still want to use the `certbot` if we update ubuntu versions, say to version `17`?